ParquetClassLibrary/Sandbox/ID/ParquetID.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Newtonsoft.Json;
+
+namespace ParquetClassLibrary.Sandbox.ID
+{
+    /// <summary>
+    /// Uniquely identifies every defined parquet.
```

Comment:
There was some initial confusion about whether or not each instance of a Parquet has a ParquetID, or if each classification of Parquet has a ParquetID. This particular block, versus this kind of block. After reading more of the code review, we were able to figure out that it's a classification, but maybe the comment here can be adjusted?

Reply:
Cool, thank you very much — that's exactly what I'm looking for!
Fixed as of aa0e7eb.